cloud/azure/database/mariadb/mode/replication.pm

```diff
@@ -0,0 +1,172 @@
+#
+# Copyright 2022 Centreon (http://www.centreon.com/)
+#
+# Centreon is a full-fledged industry-strength solution that meets
+# the needs in IT infrastructure and application monitoring for
+# service performance.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package cloud::azure::database::mariadb::mode::replication;
+
+use base qw(cloud::azure::custom::mode);
+
+use strict;
+use warnings;
+
+sub get_metrics_mapping {
+    my ($self, %options) = @_;
+
+    my $metrics_mapping = {
+        'replication_lag' => {
+            'output' => 'Replication Lag In Seconds',
+            'label'  => 'replication-lag',
+            'nlabel' => 'azmariadb.replication.lag.seconds',
+            'unit'   => 's',
+            'min'    => '0'
+        },
```

Comment:
Doesn't appear in the official doc (https://docs.microsoft.com/en-us/azure/azure-monitor/essentials/metrics-supported#microsoftdbformariadbservers). 

I think seconds_behind_master is enough.

Reply:
Yeah, for MySQL you have the servers and flexibleServer option with different metrics. This seems not to be the case for MariaDB.

Should be fixed now.